From 265026694e86009653d72a0999ccbdf950678ab8 Mon Sep 17 00:00:00 2001
From: Dmytro Laktyushkin <Dmytro.Laktyushkin@amd.com>
Date: Wed, 15 Jun 2016 17:49:10 -0400
Subject: [PATCH 0568/1722] drm/amd/dal: add otg_inst to amdgpu_crtc structure

This is  required to decouple VBLANK and PFLIP interrupt processing
from internal DC state

Signed-off-by: Dmytro Laktyushkin <Dmytro.Laktyushkin@amd.com>
Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h b/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h
index fa217346..34effd2 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h
@@ -430,6 +430,7 @@ struct amdgpu_crtc {
 	struct drm_display_mode hw_mode;
 
 	/* After Set Mode target will be non-NULL */
+	int otg_inst;
 	struct dc_target *target;
 };
 
-- 
2.7.4

